instance/models/openedx_appserver.py

```diff
@@ -660,6 +660,7 @@ def accepts_ssh_commands():
                 self.logger.info('Provisioning failed')
                 self._status_to_configuration_failed()
                 self.provision_failed_email("AppServer deploy failed: Ansible play exited with non-zero exit code", log)
+                self.manage_instance_services(active=False)
```

Comment:
There's a very similar block of code in the `except` clause on line 676, should this be added there as well?

Reply:
Good catch, @kewne. Unhandled exception definitely can cause this issue.